docs/source/advanced_usage.rst

```diff
@@ -406,21 +406,19 @@ The following shows an instructive example of all three.
 
 .. code:: python
 
-    import pyquil.paulis as pl
+    from pyquil.paulis import exponential_map
 
-    # Simplification
     sigma_cubed = sigma * sigma * sigma
-    print("Simplified  : {}".format(sigma_cubed))
-    print()
+    print(f"Simplified: {sigma_cubed}\n")
 
-    #Produce Quil code to compute exp[iX]
+    # Produce Quil code to compute exp[iX]
     H = -1.0 * sX(0)
-    print("Quil to compute exp[iX] on qubit 0:")
-    print(pl.exponential_map(H)(1.0))
+    print(f"Quil to compute exp[iX] on qubit 0:\n"
+           "{exponential_map(H)(1.0)}")
```

Comment:
I think you need to prefix the second double-quoted string with `f` as well. 

```suggestion
           f"{exponential_map(H)(1.0)}")
```

That's based on a local experiment:

    >>> x = "x"
    >>> f"y" "{x}"
    'y{x}'
    >>> f"y" f"{x}"
    'yx'